packages/components/src/SVG/SVG.tsx

```diff
@@ -1,79 +1,60 @@
 import React, { ReactNode, SVGAttributes } from "react"
 import classnames from "classnames"
 import { OverrideClassName } from "~types/OverrideClassName"
-import { warn } from "~utils/console"
 import styles from "./SVG.module.scss"
 
-type RolesType =
-  | "img" // meaningful, title should be read aloud to users who can't see it
-  | "presentation" // decorative, should be silent to users who can't see it
-
-export type SVGProps = {
-  children: ReactNode
+type BaseSVGProps = {
   inheritSize?: boolean
-  role?: RolesType
-  title?: string
-  desc?: string
```

Comment:
So we don't want these anymore right? Probably worth mentioning in the changeset that they are removed.

Reply:
I did originally but G wanted to me to treat this like a new component rather than a breaking change for the changeset https://github.com/cultureamp/kaizen-design-system/pull/3959#discussion_r1306970644

Reply:
Oh. Okay sure.